tests/unit/assessment/test_pipelines.py

```diff
@@ -21,14 +21,22 @@ def test_pipeline_assessment_with_config(mocker):
         )
     ]
 
-    ws = Mock()
+    ws = MagicMock()
```

Comment:
Can you create_autospec workspace client instead?

Reply:
Using create_autospec now